### ISSUE TITLE
PHP-FPM Bug when `aerospike` is disabled (default behavior)

### ISSUE THREAD

**Issue by Mahmoudz** [laradock/laradock#405]
Reproducing steps:
1. `docker-compose up -d php-fpm`
2. `docker-compose exec php-fpm bash`
3. `php` or any php command `php -v`

**Error Details:**

```shell
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/lib/php/extensions/no-debug-non-zts-20151012/aerospike.so' - /usr/local/lib/php/extensions/no-debug-non-zts-20151012/aerospike.so: cannot open shared object file: No such file or directory in Unknown on line 0
PHP Warning:  PHP Startup: Invalid library (appears to be a Zend Extension, try loading using zend_extension=opcache.so from php.ini) in Unknown on line 0
```

After setting Aerospike to true in the docker-compose file and rebuild I get this warning:

```
PHP Warning:  PHP Startup: Invalid library (appears to be a Zend Extension, try loading using zend_extension=opcache.so from php.ini) in Unknown on line 0
```

**Comment by gagarine** [comment]
I got somthings similar when using xdebugPhpFpm

```
➜  laradock git:(master) ✗ ./xdebugPhpFpm start
Start xDebug
laradock_php-fpm_1
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/lib/php/extensions/no-debug-non-zts-20151012/aerospike.so' - /usr/local/lib/php/extensions/no-debug-non-zts-20151012/aerospike.so: cannot open shared object file: No such file or directory in Unknown on line 0
PHP Warning:  PHP Startup: Invalid library (appears to be a Zend Extension, try loading using zend_extension=opcache.so from php.ini) in Unknown on line 0
PHP 7.0.9 (cli) (built: Aug 10 2016 19:45:48) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.9, Copyright (c) 1999-2016, by Zend Technologies
    with Xdebug v2.5.0, Copyright (c) 2002-2016, by Derick Rethans
```

**Comment by mikeerickson** [comment]
@gagarine @mahmoudz I have seen this quite often over the weekend working on the xDebug stuff on macOS. I will tackle this issue after I am done with xDebug tasks.

**Comment by alchimik** [comment]
fix for aerospike - #622
fix for opcache - #617
